src/app/modules/webhook/webhook.utils.ts

```diff
@@ -19,6 +20,13 @@ export const validateWebhookUrl = (webhookUrl: string): Promise<any> => {
       )
     }
     const urlParsed = new URL(webhookUrl)
+    if (urlParsed.origin === config.app.appUrl) {
```

Comment:
is it better and safer to compare strict equality after parsing the [URL origin](https://nodejs.org/api/url.html#url_url_origin) instead?

Reply:
@liangyuanruo as in something like this? 
```
const webhookUrlParsed = new URL(webhookUrl)
const appUrlParsed = new URL(config.app.appUrl)
if (webhookUrlParsed.origin === appUrlParsed.origin) { ... }
```